app/models/service_template.rb

```diff
@@ -242,4 +242,8 @@ def validate_template
       r.respond_to?(:template_valid?) && !r.template_valid?
     end.try(:resource).try(:validate_template) || {:valid => true, :message => nil}
   end
+
+  def validate_order
+    service_template_catalog && display
```

Comment:
@abellotti Does this method get dynamically called based on adding the `validate_action` in `config/api.yml`?

I was thinking this would be `orderable?` and could create an alias to `validate_order`.  Would also like to put some tests around this either here or a followup PR.